raiden/exceptions.py

```diff
@@ -149,7 +149,7 @@ class TokenNotRegistered(RaidenError):
     """ Raised if there is no token network for token used when opening a channel  """
 
 
-class AlreadyRegisteredTokenAddress(RaidenError):
+class AlreadyRegisteredTokenAddress(RaidenRecoverableError):
```

Comment:
It should be a `RaidenValidationError` as explained in https://github.com/raiden-network/raiden/issues/4878#issuecomment-534093487